client/signup/controller.js

```diff
@@ -84,6 +85,7 @@ export default {
 		} else if (
 			context.pathname.indexOf( 'domain' ) >= 0 ||
 			context.pathname.indexOf( 'plan' ) >= 0 ||
+			context.pathname.indexOf( 'onboarding-new' ) >= 0 ||
```

Comment:
What does `onboarding-new` refer to?

We have `/new` onboarding flow so I'm worried this is confusing.

Reply:
I've renamed `onboarding-new` to `onboarding-registrationless`